jira/autokitteh.yaml

```
@@ -0,0 +1,14 @@
+version: v1
+
+project:
+  name: jira_to_calendar_sync
+  connections:
+    - name: my_jira
+      integration: jira
+    - name: my_googlecalendar
```

Nit (more readable) - and don't forget to change the `.py` code  accordingly.

```suggestion
    - name: my_google_calendar
```